spring-boot-project/spring-boot/src/test/java/org/springframework/boot/logging/logback/LogbackLoggingSystemTests.java

```diff
@@ -137,6 +140,8 @@ public void withFile() throws Exception {
 	public void testBasicConfigLocation() throws Exception {
 		this.loggingSystem.beforeInitialize();
 		assertThat(getConsoleAppender()).isNotNull();
+		assertThat(ManagementFactory.getPlatformMBeanServer().queryMBeans(
```

Comment:
Please test this in a dedicated test. I'd like also to see a test that checks what happens if the MBean has already been registered (using the `<jmxConfigurator />` option).